src/SFA.DAS.FAT.Web/Models/CoursesProviderViewModel.cs

```diff
@@ -29,6 +34,26 @@ public bool IsDayReleaseAvailable
         get => Locations.Any(l => l.DayRelease);
     }
 
+    public bool IsBlockReleaseMultiple
+    {
+        get => Locations.Count(x => x.BlockRelease) > 1;
+    }
+
+    public bool IsDayReleaseMultiple
+    {
+        get => Locations.Count(x => x.DayRelease) > 1;
+    }
+
+    public decimal? NearestBlockRelease
+    {
+        get => Locations.FirstOrDefault(x => x.BlockRelease)?.CourseDistance;
+    }
+
+    public decimal? NearestDayRelease
+    {
+        get => Locations.FirstOrDefault(x => x.DayRelease)?.CourseDistance;
+    }
```

Comment:
You are doing FirstOrDefault in multiple places, this being linq statement it is not very performant. Might as well just do it once in the operator and populate all the needed bits in the operator. 
Also I cannot comment on existing code, so all the properties which are populated in the operator can be markes init instead of set. Or just mage readonly.

Reply:
i've rewritten this to make it all happen in an operator, and same for comment below.